packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -82,6 +82,7 @@ type TReduxProps = {
   searchUrl: null | string;
   trace: FetchedTrace | TNil;
   uiFind: string | TNil;
+  graphLayoutManagerMemory: number | undefined;
```

Comment:
I'm not an expect in React, so a Q: why does this setting need to be in the properties (which causes more code changes) instead of only passed from config to `<TraceGraph>`? Or even looked up from config by `TraceGraph` directly?

Reply:
I haven't worked on React for a while, so I might be wrong.


1>
I observed in this repo:
* The config is passed into React components via Redux.
* Only top/page level components (like TracePage in this example) subscribe to Redux, and most sub-components don't.

So I am trying to follow the same style here - have `TracePage` get config from redux and pass to `TraceGraph` via property.

2>
Since we let component `TracePage` subscribe to Redux, then we need to make sure what we need from Redux is exposed as component property by `mapStateToProps` function defined in the component.

We can either expose the entire config object, or sub fields/objects,  but either requires code change - adding into component property definition.

The reason I exposed `graphLayoutManagerMemory` (now I changed to `traceGraphConfig`) instead of entire config object is due component re-render.
If any property value of the component changes, it causes component to re-render. The entire config object contains other fields/sub objects that current component doesn't need at all, which might cause unnecessary re-render.